src/Collections/TerminusCollection.php

```diff
@@ -262,7 +277,7 @@ public function getMemberList($key = 'id', $value = 'name')
      */
     protected function getMembers()
     {
-        if (empty($this->models)) {
+        if ($this->models === null) {
```

Comment:
😆 

In this case what I'm doing is distinguishing between an empty collection, and one which has not yet been fetched. It might be a slightly subtle use of PHP type, but previously, the initial state of a collection was the same as a fetched but empty collection which introduced an even more subtle bug.

The alternative is to store the 'loaded' state in an explicit variable rather than implicitly using the type of the model list. But type safety is for wimps. This is PHP. Viva la dynamism!